exporting/tests/exporting_fixtures.c

```diff
@@ -127,3 +127,35 @@ int teardown_initialized_engine(void **state)
 
     return 0;
 }
+
+int setup_prometheus(void **state)
+{
+    (void)state;
+
+    prometheus_exporter_instance = calloc(1, sizeof(struct instance));
```

Comment:
You are using `calloc` without to verify the return, please, add a test here.

Reply:
No need to test the result. The code is for unit tests only and cmocka substitutes it's own `calloc` here.
```
/* Use the real malloc in this function. */
#undef malloc
void* _test_malloc(const size_t size, const char* file, const int line) {
...
block = (char *)malloc(allocate_size);
    assert_non_null(block);
...
```

Reply:
All right, thank you!